magellan-support/src/main/java/com/wealthfront/magellan/support/SingleActivity.java

```diff
@@ -34,6 +34,7 @@ protected void onPostCreate(@Nullable Bundle savedInstanceState) {
   @Override
   public void onSaveInstanceState(Bundle outState) {
     navigator.onSaveInstanceState(outState);
+    super.onSaveInstanceState(outState);
```

Comment:
Sneaky fix for #42 (also fixes a new lint error)

Reply:
I'll take it